.travis.yml

```diff
@@ -2,6 +2,7 @@ language: ruby
 script:
   - bundle exec rspec
   - bundle exec middleman build
+cache: bundler
```

Comment:
I don't think this is related to this work?

Also, according to the [Travis documentation](https://docs.travis-ci.com/user/languages/ruby/#Caching-Bundler) this only takes effect on private projects.

Reply:
Yes, it seems like it doesn't affect our project.